templates/frontend/index.html

```diff
@@ -10,21 +10,41 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <meta name="description" content="Django + React + Webpack boilerplate used and made by Labcodes" />
+  <meta name="description" content="Django + React + Webpack boilerplate used and made by Labcodes"/>
```

Comment:
```suggestion
  <meta name="description" content="Django + React boilerplate used and made by Labcodes"/>
```

Reply:
There is some other places where we still mention Weback like in the `Header.jsx`